testgrid/web/env/development.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   ENVIRONMENT: "development",
-  API_ENDPOINT: "https://api.testgrid.kurl.sh/api/v1",
```

Comment:
was this intended to be committed?

Reply:
Yes it was unintentional the first time it was changed

https://github.com/replicatedhq/kURL/pull/3369/commits/84356793ecc2a8644734efd607ae6a920162884e